### PR TITLE
Adding VSHNMinio user alerting

### PR DESCRIPTION
**Pull request by wejdross**
Adding VSHNMinio user alerting

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

### CODE REVIEW

**Comment by wejdross** [comment]
appcat version will be adjusted after merging: https://github.com/vshn/appcat/pull/178

**Review by zugao — APPROVED** [review]
LGTM. We have to update the end user docs as well.

**Comment by wejdross** [comment]
> LGTM. We have to update the end user docs as well.

https://docs.appcat.ch/vshn-managed/minio.html
VSHNMinio is the only service managed by us, there are no user docs for that.

File: component/class/defaults.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ parameters:
       appcat:
         registry: ghcr.io
         repository: vshn/appcat
-        tag: v4.75.0
+        tag: minioAlerting
```
Comment (Kidswiss) [inline component/class/defaults.yml:52]: Don't forget to adjust it once appcat itself is tagged

**Review by Kidswiss — APPROVED** [review]

**Pull request merged**